### PR TITLE
Add labels parameter

### PR DESCRIPTION
**Pull request by Bolodya1997**
# Issue
Currently ICMP responder doesn't allow user to add some custom labels.
# Solution
Add labels to the environment configuration.

### CODE REVIEW

File: internal/pkg/imports/gen.go [hunk 1]
```diff
@@ -14,7 +14,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Package imports is used for generating list of imports to optimize use of docker build cache
 package imports
 
+//go:generate bash -c "rm -rf imports_linux.go"
```
Comment (denis-tingaikin) [inline internal/pkg/imports/gen.go:20]: Do we really need this?

**Review by denis-tingaikin — APPROVED** [review]
Let's figure out why tests fail here and after that, it can be merged.

File: internal/pkg/imports/gen.go (hunk 1 above)
Comment (Bolodya1997) [reply internal/pkg/imports/gen.go:20]: Yep, without it we can get such situation:
1. `a.go` imports `foo/bar`
2. we generate imports -> now `imports.go` imports `foo/bar`
3. we delete `a.go` -> so now only `imports.go` imports `foo/bar`
4. we generate imports -> `imports.go` still imports `foo/bar`

File: suite_test.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	cancel     context.CancelFunc
+	x509source x509svid.Source
+	x509bundle x509bundle.Source
+	config     main.Config
+	spireErrCh <-chan error
+	sutErrCh   <-chan error
+	sutCC      grpc.ClientConnInterface
+}
+
+func TestForwarderTestSuite(t *testing.T) {
```
Comment (denis-tingaikin) [inline suite_test.go:45]: Let's rename this

Comment (Bolodya1997) [reply suite_test.go]: Oops, fixed :)

**Review by denis-tingaikin — APPROVED** [review]

File: internal/pkg/imports/gen.go (hunk 1 above)
Comment (denis-tingaikin) [reply internal/pkg/imports/gen.go:20]: I think step 3 is incorrect, you need to add step to run go mod tidy.

**Pull request merged**